apps/judicial-system/web/src/components/InfoCard/useInfoCardItems.tsx

```diff
@@ -364,6 +365,42 @@ const useInfoCardItems = () => {
       : [],
   }
 
+  const victims: Item = {
+    id: 'victim-item',
+    title: (
+      <Text
+        variant="h4"
+        as="h4"
+        marginBottom={
+          workingCase.victims && workingCase.victims.length > 1 ? 3 : 2
+        }
+      >
+        {capitalize(
```

Comment:
The use of `capitalize` here is not needed since we have hardcoded strings that are already capitalized.